api/routes/users/put.js

```diff
@@ -68,7 +68,7 @@ module.exports = (
           logger.verbose(`state [${req.body.state}] is invalid`);
           return res
             .status(400)
-            .send('update-user-invalid-state')
+            .send({ error: 'edit-account.invalid-state' })
```

Comment:
And for some reason this goofball method just sent strings instead of whole objects.  :sigh: